src/modules/localStorage.js

```diff
@@ -0,0 +1,10 @@
+function getTasks() {
+  const tasks = localStorage.getItem('tasks');
+  return tasks ? JSON.parse(tasks) : [];
+}
+
+function saveTasks(tasks) {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
```

Comment:
In this part of your code, I suggest you use arrow functions to follow JavaScript best practices for the ES6 approach.